src/string-class.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-extend-native */
+/* eslint-disable arrow-body-style */
+/* eslint-disable prefer-template */
+
+String.prototype.hasVowels = function hasVowels() {
+  /* This method returns true if there are vowels
+     in a string and false for absence of vowels
+  */
+  const vowels = /[aeiou]/i;
+  return vowels.test(this);
+};
+
+String.prototype.toUpper = function toUpper() {
+  /* Converts a string to uppercase format */
+  const lower = /[a-z]/g;
+  return this.replace(lower, (letter => {
+    return String.fromCharCode(letter.charCodeAt(0) - 32);
```

Comment:
No need to get the `index 0` since letter on it's own will always be at index 0.